botnim/vector_store/vector_store_es.py

```diff
@@ -1,9 +1,10 @@
 import os
```

Comment:
Also in this file you undid all the changes I made.

Reply:
@akariv fixed?